### PR TITLE
Don't _require_ CNS be activated

### PR DESCRIPTION
**Pull request by misterbisson**
And fix other nits from https://github.com/joyent/triton-docker-cli/pull/9

### CODE REVIEW

File: triton-docker [hunk 1]
```diff
@@ -68,11 +68,28 @@ function currentProfile () {
...
     fi
 
-    echo "Executing in '$(triton profile get | awk 'NR == 1 {print $2}')' (${profile_source})"
+    # Get and the user's account information and CloudAPI URL for parsing later
+    local triton_account="$(triton account get)"
+
+    echo "Executing in '$(echo "${triton_account}" | awk 'NR == 1 {print $2}')' (${profile_source})"
+
+    # Check if CNS is enabled, require it
+    if [ ! "true" == "$(echo "${triton_account}" | awk -F": " '/cns/{print $2}')" ]
```
Comment (trentm) [inline triton-docker]: That is brittle. If I have "cns" in my email or name or other fields:

```
$ triton account get| awk -F": " '/cns/{print $2}'
trent.mick+cns-guy@joyent.com
true
```

Perhaps:

```
... | awk -F": " '/^triton_cns_enabled:/{print $2}'
```

File: triton-docker [hunk 2]
```diff
@@ -68,11 +68,28 @@ function currentProfile () {
...
+        tput rev  # reverse
+        tput bold # bold
+        echo 'Error! Triton CNS is not enabled.'
+        tput sgr0 # clear
+        echo
+        echo 'Please run the following to activate:'
+        echo '  triton account update triton_cns_enabled=true'
+        echo
+    fi
 }
```
Comment (trentm) [inline triton-docker:89]: You want this to be an error? Rather than a warning?

If an *error* it should exit non-zero.

File: triton-docker [hunk 3]
```diff
@@ -85,27 +102,11 @@ function CnsEnvVars () {
...
-        tput sgr0 # clear
-        echo
-        echo 'Please run:'
-        echo '  triton account update triton_cns_enabled=true'
-        echo
-        exit 1
-    fi
-
     # Get the user's UUID
     local triton_account_uuid="$(echo "${triton_account}" | awk -F": " '/id:/{print $2}')"
```
Comment (trentm) [inline triton-docker]: perhaps `/^id:`

File: triton-docker (hunk 2 above)
Comment (misterbisson) [reply triton-docker:89]: Changed to a warning message.

File: triton-docker (hunk 3 above)
Comment (misterbisson) [reply triton-docker]: Fixed.

File: triton-docker (hunk 1 above)
Comment (misterbisson) [reply triton-docker]: Fixed.

**Pull request merged**